contribuer/public/admin/config.yml

```
@@ -18,6 +18,40 @@ publish_mode: editorial_workflow
 root: ../../..
 
 fields:
+  field_boolean: &field_boolean
+    widget: object
+    fields:
+      - label: Valeur
+        name: value
+        widget: boolean
+  field_number: &field_number
+    widget: object
+    fields:
+      - label: Opérateur
+        name: operator
+        widget: select
+        options:
+          - label: Supérieur à
+            value: ">"
+          - label: Supérieur ou égal à
+            value: ">="
+          - label: Égal à
+            value: "==="
```

```suggestion
            value: "="
```

Pour coller à une logique moins technique et plus généraliste.

---

Ok pour le `=`

> D'ailleurs cette condition est-elle vraiment nécessaire ? Aucune aide n'en a besoin (pour le moment).
Non mais c'est une simplification qui n'ajoute pas de complexité et qui réduit la complexité et le potentiel blocage des utilisateurs.
Ex pass_culture, il faut avoir strictement 18 ans

---

Je test mon changement demain